output/opendap_post2.sh

```diff
@@ -21,6 +21,7 @@
 #------------------------------------------------------------------------
 #
 THIS=$(basename -- $0)
+APPLOG=$THIS.log
```

Comment:
Can you explain what changes are made here and what `$APPLOG` is meant to be? I am definitely not against better logging, but is this creating an new log file type? E.g., will support for something like `tailf applog` be needed.

Reply:
Yes I am diverting the log messages from this script into their own file instead of sending them to the main log file. There are a lot of them and they seem to be filling up that file and making it difficult to read. The messages are really only useful for troubleshooting so it seems that they could be checked manually (rather than using something like `tailf opendap_post.sh.log`)

Reply:
@jasonfleming that works for me!